ivy/functional/frontends/torch/creation_ops.py

```diff
@@ -0,0 +1,11 @@
+# local
+import ivy
+
+
+def ones(size, *, out=None, dtype=None, layout=None, device=None, requires_grad=False):
```

Comment:
we should actually handle `requires_grad`, we can do this by calling `ivy.variable` on the returned value if `requires_grad=True`